oidc-handler/pom.xml

```diff
@@ -1,11 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.apache.sling</groupId>
+  <artifactId>org.apache.sling.oidchandler</artifactId>
```

Comment:
I think the artifactId should include 'auth' somewhere, to fall in line with our other modules: auth-core, auth-form, auth-xing-oauth. Same goes for the name.